src/main/java/teammates/common/datatransfer/questions/FeedbackConstantSumQuestionDetails.java

```diff
@@ -51,6 +77,10 @@ public FeedbackConstantSumQuestionDetails(String questionText) {
         this.points = 100;
         this.forceUnevenDistribution = false;
         this.distributePointsFor = FeedbackConstantSumDistributePointsType.NONE.getDisplayedOption();
+        this.hasMinPoint = false;
+        this.hasMaxPoint = false;
+        this.maxPoint = null;
+        this.minPoint = null;
```

Comment:
Could I ask if `hasMaxPoint` and `hasMinPoint` are really needed, because it seems we could get hasMaxPoint from checking the value of maxPoint == null?

Reply:
Thanks for pointing this out! Will refactor!